src/spring-cloud/azext_spring_cloud/_transformers.py

```diff
@@ -5,6 +5,7 @@
 
 
 # pylint: disable=line-too-long
+from msrestazure.tools import parse_resource_id
```

Comment:
Use "from azure.mgmt.core.tools import parse_resource_id" instead as "msrestazure.tools" will be deprecated soon.

Reply:
Updated, as well as import in other files